steps/http/steps.go

```diff
@@ -61,6 +61,10 @@ func (s Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioConte
 		}
 		return session.ConfigureRequestBodyJSONProperties(ctx, props)
 	})
+	scenCtx.Step(`^the HTTP request body based in the JSON$`, func(message *godog.DocString) error {
```

Comment:
I'd rename it to `^the HTTP request body with the JSON$`

Reply:
Done